cookbooks/aws-parallelcluster-environment/recipes/config/network_interfaces.rb

```diff
@@ -88,7 +88,8 @@ def cidr_to_netmask(cidr)
         'DEVICE_IP_ADDRESS' => device_ip_address,
         'CIDR_PREFIX_LENGTH' => cidr_prefix_length,
         'NETMASK' => netmask,
-        'CIDR_BLOCK' => cidr_block
+        'CIDR_BLOCK' => cidr_block,
+        'MAC' => mac
```

Comment:
why the MAC address became required just now with AL2023?

Reply:
MAC is required as part of the configuration file

Reply:
Yep, but this is executed for all OSes not only on AL2023, right? 
So why are we injecting it now also for other OSes?

Reply:
Yes, but scripts of other OSes does not use the variable. So behavior of other scripts is not changed

Reply:
It's not a good approach to inject unneeded envars. Since we are on a rush, please simply add a TOFIX comment saying that that envar should be injected only for AL23.

Reply:
Done